roboto/api_types.py

```diff
@@ -267,34 +389,130 @@ class SuccessfulPayment:
 
 
 @dataclass(frozen=True)
-class MessageBase:
-    """Base data for a Telegram message."""
+class PassportData:
+    """Contains information about
```

Comment:
Please keep the same wording style as previous docstrings.

Also, the format of docstrings should follow the convention of having a summary consisting of a single line, and then, if more information is needed, it should be on a paragraph separated by a blank line. The closing quotes should be on their own line if this is the case:

```
"""An example docstring.

Every other information that may be needed comes here,
so that the user can read it if needed.
"""
```